compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.td

```diff
@@ -187,6 +186,37 @@ def FormDispatchWorkgroupsPass :
   ];
 }
 
+def DispatchTensorPass :
```

Comment:
I think a better name of this is `TensorToFlowPass` ?

Reply:
Unresolving this. `DispatchTensorPass` is not very descriptive of what this pass is doing.

Reply:
+1 on TensorToFlow